src/components/Sidenav/SidenavBase.style.ts

```diff
@@ -29,7 +29,7 @@ type SidebarNavLinkProps = {
 export const SidebarNav = styled.nav<SidebarNavProps>`
   position: fixed;
   top: 0;
-  bottom: 0;
+  height: 100vh;
```

Comment:
I think this could also work any may be a bit more safer?

```suggestion
  bottom: 0;
  height: 100%;
```